IdentityCore/src/cache/crypto/MSIDAssymetricKeyPair.m

```diff
@@ -47,6 +47,94 @@ - (nullable instancetype)initWithPrivateKey:(SecKeyRef)privateKey
     return self;
 }
 
+- (NSString *)getKeyExponent:(SecKeyRef)keyRef
```

Comment:
would be great to have unit tests for this logic

Reply:
Will add unit tests for this in a separate PR.